pkg/kubemark/hollow_kubelet.go

```diff
@@ -67,7 +67,7 @@ func NewHollowKubelet(
 	volumePlugins := empty_dir.ProbeVolumePlugins()
 	volumePlugins = append(volumePlugins, secret.ProbeVolumePlugins()...)
 	d := &kubelet.Dependencies{
-		KubeClient:        client,
```

Comment:
if this fails as expected, you might try simulating the issue the other PR caused (the "" dockershim)

Reply:
Yup... I was planning to do that. Before that wanted to check if it's something obvious with testinfra

Reply:
So it failed as expected. Trying now with the revert reverted.